python/private/py_cc_toolchain_info.bzl

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation of PyCcToolchainInfo."""
+
+PyCcToolchainInfo = provider(
```

Comment:
Can we generate the documentation for this API and include the docs in the docs folder?  Are we marking this API GA or Beta?  I prefer Beta until we validate that the API is stable and won't change.

Reply:
Yeah, definitely beta for now.

I'll see how well these show up as generated docs. I think the fronting macro is going to ruin it and the docs will have to be copy/pasted.

Reply:
Docs generated

Reply:
Can we mark in the docs that the API is beta and may change?